src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -84,6 +95,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $filenames = (array) $input->getArgument('filename');
+        $excludes = explode(' ', $input->getOption('exclude'));
```

Comment:
We can configure the option instead to accept multiple arguments using `InputOption::VALUE_IS_ARRAY`.

Reply:
Changed :+1: